recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -190,6 +190,30 @@ install:
           # DEBUG_MESSAGE is used to collect all necessary debug info we need.
           DEBUG_MESSAGE=""
 
+          # Determine if user is installing in gke autopilot
+          while :; do
+            echo -e -n "${GREEN}Are you installing in a GKE Autopilot cluster? Y/N? ${NC} "
```

Comment:
I think you're going to have to add an
`if [[ "{{.NEW_RELIC_ASSUME_YES}}" == "true" ]]; then
                  GKE_AUTOPILOT_ANSWER="N"
`
somewhere in this conditional, its for when the install process allows you to set certain flags, to answer all the questions asked in the install process. I think they use this flag in some automated testing suites and it'll break if theres no default answer defined. I'm going to assume that we're probably going to have to assume that gke autopilot is not the default mode, so answer is 'N' not 'Y' but give it a thought and see what you think 

https://github.com/newrelic/newrelic-cli/blob/main/docs/cli/newrelic_install.md 

take a look and see what you think about the default answer

Reply:
Done